src/ioctl.c

```diff
@@ -249,8 +249,9 @@ static long scoutfs_ioc_ino_path(struct file *file, unsigned long arg)
 		ret = 0;
 	}
 
-out:
+out_free:
 	scoutfs_dir_free_backref_path(sb, &list);
```

Comment:
I don't think this is needed because the list is initialized and empty.  It's always safe to try and free it.